docs/tutorial/beginner.rst

```diff
@@ -0,0 +1,439 @@
+##################################################
+Beginner Tutorial - Set up & Running One Algorithm
+##################################################
+
+TODO: add an explanation of this tutorial
+
+Step 0: Clone the SPRAS repository, set up the environment, and run Docker
+==========================================================================
+
+0.1 Start Docker
+----------------
+
+Launch Docker Desktop and wait until it says "Docker is running".
+
+0.2 Clone the SPRAS repository
+-------------------------------
+
+Visit the `SPRAS github repository <https://github.com/Reed-CompBio/spras>`__ and clone it locally
+
+0.3 Set up the SPRAS environment
+-------------------------------------
+
+From the root directory of the SPRAS repository, create and activate the Conda environment:
```

Comment:
And install the SPRAS Python package (because that is what the commands will do)